src/main/java/co/rsk/federate/signing/hsm/advanceblockchain/ConfirmedBlockHeadersProvider.java

```diff
@@ -33,10 +33,10 @@ public ConfirmedBlockHeadersProvider(
         this.hsmVersion = hsmVersion;
     }
 
-    public List<BlockHeader> getConfirmedBlockHeaders(Keccak256 startingPoint) {
-        List<BlockHeader> potentialConfirmed = new ArrayList<>();
+    public List<Block> getConfirmedBlockHeaders(Keccak256 startingPoint) {
```

Comment:
Why not change the method name as well to `getConfirmedBlocks`?

Reply:
Updated